csharp/downgrades/83aca6b3e4fa38dd2b97b9b51dfc199a2ba9c7f2/upgrade.properties

```diff
@@ -0,0 +1,2 @@
+description: Remove list- and slice pattern expression kinds.
+compatibility: backwards
```

Comment:
Is this downgrade batch correct? Shouldn't the unsupported expression types be changed to `@unknown_expr`?